homeassistant/components/azure_data_explorer/client.py

```diff
@@ -45,24 +52,36 @@ def __init__(self, data: Mapping[str, Any]) -> None:
             ingestion_mapping_reference="ha_json_mapping",
         )
 
-        # Create cLient for ingesting and querying data
-        kcsb = KustoConnectionStringBuilder.with_aad_application_key_authentication(
-            self._cluster_ingest_uri,
-            data[CONF_APP_REG_ID],
-            data[CONF_APP_REG_SECRET],
-            data[CONF_AUTHORITY_ID],
+        # Create client for ingesting data
+        kcsb_ingest = (
+            KustoConnectionStringBuilder.with_aad_application_key_authentication(
+                self._cluster_ingest_uri,
+                data[CONF_APP_REG_ID],
+                data[CONF_APP_REG_SECRET],
+                data[CONF_AUTHORITY_ID],
+            )
+        )
+
+        # Create client for querying data
+        kcsb_query = (
+            KustoConnectionStringBuilder.with_aad_application_key_authentication(
+                self._cluster_query_uri,
```

Comment:
```suggestion
                data[CONF_ADX_CLUSTER_INGEST_URI].replace("ingest-", ""),
```

Is `ingest-` only at the start or the URI or also in the middle?
If only at the start we should use `removeprefix` instead

Reply:
@kaareseras Can you please answer my question?

Reply:
Sorry, didnt see it. the URL is always https://ingest-xxxxxxxx